README.md

```diff
@@ -78,305 +121,522 @@ SendBirdCall.connectWebSocket()
 
-> The media viewers can also be set using `call.setLocalMediaView(element)` or `call.setRemoteMediaView(element)`.  
+> The media viewers can also be set using the `call.setLocalMediaView(element)` or `call.setRemoteMediaView(element)`.  
 
 ```html
 <video id="remote_video_element_id" autoplay>
 ```
 
 ```javascript
-//to set media viewer lazily
+// Set media viewer lazily
 call.setLocalMediaView(document.getElementById('local_video_element_id'));
 call.setRemoteMediaView(document.getElementById('remote_video_element_id'));
 ```
 
-## Receive a call
-Receive incoming calls by first registering a listener. Accept or decline incoming calls using the `call.accept()` or the `call.end()` methods. If the call is accepted, a media session will automatically be established.
+### Step 5: Receive a call
+
+Register a listener first to receive incoming calls. Accept or decline incoming calls by using the `call.accept()` or the `call.end()` methods. If the call is accepted, a media session will automatically be established.
+
+Before accepting any calls, event listeners such as `onEstablished`, onConnected must be registered. Once registered, these listeners enable reacting to in-call events through callbacks methods.
+
+```javascript
+SendBirdCall.addListener(UNIQUE_HANDLER_ID, {
+    onRinging: (call) => {
+        call.onEstablished = (call) => {
+            ...
+        };
+        
+        call.onConnected = (call) => {
+            ...
+        };
+        
+        call.onEnded = (call) => {
+            ...
+        };
+        
+        call.onRemoteAudioSettingsChanged = (call) => {
+            ...
+        };
+        
+        call.onRemoteVideoSettingsChanged = (call) => {
+            ...
+        };
+        
+        const acceptParams = {
+            callOption: {
+                localMediaView: document.getElementById('local_video_element_id'),
+                remoteMediaView: document.getElementById('remote_video_element_id'),
+                audioEnabled: true,
+                videoEnabled: true
+            }
+        };
+        
+        call.accept(acceptParams);
+    }
+});
+```
+
+> **Note**: If the media viewer elements have been set through the `call.setLocalMediaView()` and `call.setRemoteMediaView()`, ensure that the same media viewers are set in `acceptParams`’s `callOption`. Otherwise, these will be overridden during the `call.accept()`.
+
+Incoming calls are received through the application's persistent internal server connection, which is established by `SendBirdCall.connectWebSocket()`. In the event of accidental disconnection, the application will attempt to reconnect every 2 seconds.
+
+<br />
+
+## Implementation guide
```

Comment:
What's the difference between `Implementation guide` and `Making your first call`? Much of these two sections seem duplicated.

Reply:
We know about this, but we think it's okay to keep the duplicated content in the `Implementation guide` section because a visitor can directly move to the section according to their own needs without seeing `Making your first call` section. 

When we all can start a total renewal of this README.md, let's discuss this.